WORKING_GROUPS.md

```diff
@@ -1,11 +1,11 @@
 # Node.js Working Groups
 
 Node.js Working Groups are autonomous projects created by the
-[Technical Steering Committee (TSC)](https://github.com/nodejs/node/blob/master/GOVERNANCE.md#technical-committee).
+[Core Technical Committee (CTC)](https://github.com/nodejs/node/blob/master/GOVERNANCE.md#core-technical-committee).
```

Comment:
Hmmm, if I understand correctly CTC is purely a technical body. So this should be TSC only I guess